plugins/node/opentelemetry-instrumentation-oracledb/src/utils.ts

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright (c) 2025, Oracle and/or its affiliates.
+ * */
+import { safeExecuteInTheMiddle } from '@opentelemetry/instrumentation';
+import {
+  Span,
+  SpanStatusCode,
+  Tracer,
+  context,
+  SpanKind,
+  trace,
+  diag,
+} from '@opentelemetry/api';
+import {
+  SEMATTRS_DB_CONNECTION_STRING,
+  ATTR_SERVER_PORT,
+  ATTR_SERVER_ADDRESS,
+  SEMATTRS_NET_TRANSPORT,
+  SEMATTRS_DB_USER,
+  SEMATTRS_DB_STATEMENT,
+} from '@opentelemetry/semantic-conventions';
+import {
+  ATTR_DB_SYSTEM,
+  ATTR_DB_NAMESPACE,
+  ATTR_DB_OPERATION_NAME,
+} from './semconv';
+import * as oracledbTypes from 'oracledb';
```

Comment:
```suggestion
import type * as oracledbTypes from 'oracledb';
```

Reply:
The type script [definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/oracledb) still does not have new exported class [`TraceHandlerBase`](https://github.com/oracle/node-oracledb/blob/main/lib/traceHandler.js)  , Hence importing both types and values. 


Should I keep it for now and remove as types definitions get updated?

Reply:
The TraceHandlerBase is modelled as abstract class, hence I need to extend and not implement. I made changes to use dynamic require with a function, Please see and let me know if this is fine